dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/FATSuite.java

```diff
@@ -75,9 +75,9 @@ public class FATSuite {
         }
     }
 
-    public static final String MOJARRA_API_IMP = "publish/files/mojarra/javax.faces-2.3.9.jar";
-    public static final String MYFACES_API = "publish/files/myfaces/myfaces-api-2.3.9.jar";
-    public static final String MYFACES_IMP = "publish/files/myfaces/myfaces-impl-2.3.9.jar";
+    public static final String MOJARRA_API_IMP = "publish/files/mojarra/javax.faces-2.3.10.jar";
```

Comment:
This should be under another issue, as we're only updating MyFaces, not Mojarra.   We can create a separate issue to update the Mojarra jars.

Reply:
I'm ok with not updating Mojarra here.